app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,29 @@
+<main>
+<div class="sign-up container">
+<h2>Sign up</h2>
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+  <div class="field">
+    <%= f.text_field :name, placeholder: "Enter your name here" %>
+  </div>
+  <div class="field">
+    
+    <%= f.email_field :email, autofocus: true, autocomplete: "email", placeholder: "Enter your email" %>
+  </div>
+  <div class="field">
+    
+    <% if @minimum_password_length %>
+    <em>(<%= @minimum_password_length %> characters minimum)</em>
+    <% end %><br />
+    <%= f.password_field :password, autocomplete: "new-password", placeholder: "Enter new password" %>
+  </div>
+  <div class="field">
+    <%= f.password_field :password_confirmation, autocomplete: "new-password", placeholder: "Confirm password" %>
```

Comment:
- [ ] [OPTIONAL] optionally you could make the login and signup page provided by devise to match the wireframe design, this will also improve your app's UI

Reply:
@PinkMoon25 Just made all changes. Thanks for you insightful review.